medical_pricelist/gql_mutations.py

```diff
@@ -39,6 +40,23 @@ class ServicesPricelistInputType(ItemsOrServicesPricelistInputType):
 def create_or_update_pricelist(
     data, user, pricelist_model, service_or_item_model, detail_model
 ):
+    incoming_name = data['name']
+    try:
+        current_service_or_item = pricelist_model.objects.get(uuid=data['uuid'])
+        current_name = pricelist_model.objects.get(uuid=data['uuid']).name
+    except KeyError:
+        current_service_or_item = None
+        current_name = None
```

Comment:
```suggestion
   pricelist_uuid = data.pop("uuid", None)
   pricelist = pricelist_model.objects.first(uuid=pricelist_uuid)
   current_name = pricelist.name if pricelist else None
```

First two parameters are moved from the latter part of the this function (lines 68 and 75). Additionally it is better to use dict.get(param) instead of catching KeyError. We also shouldn't call Model.objects.get() twice to get same object.